src/lib/util/test-utils.ts

```diff
@@ -40,3 +40,28 @@ export function checkWizardState(
   expect(wizard.completed).toBe(wizardCompleted,
     `expected wizard ${wizardCompleted ? 'to be completed' : 'not to be completed'}`);
 }
+
+
+/**
+ * Check which wizard steps are navigable using the navigation bar
+ *
+ * @param wizard Wizard component under test
+ * @param selectedStepIndex Expected selected step index
+ * @param navigableStepIndexes Array of step indexes expected to be navigable using the navigation bar
+ */
+export function checkWizardNavigableState(
```

Comment:
I'm not sure if this is a fitting name for the method. I think something like `checkPossibleDestinationSteps(...)` more fitting, because we're not really checking the state of the wizard, right?

Reply:
Agree. How about `chechWizardNavigableSteps`?

Reply:
Sounds good!

Reply:
Done in a3998c5c